{"name": "test_all_correct_except_invalid_last_name_data", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'הפרטים נקלטו בהצלחה' == 'נא להזין את השם המשפחה הנכון'\n  - נא להזין את השם המשפחה הנכון\n  + הפרטים נקלטו בהצלחה", "trace": "self = <Web.Tests.test_contactus.TestContactUs object at 0x000002BCBF1B5F90>\n\n    @allure.description(\"Test invalid last name\")\n    def test_all_correct_except_invalid_last_name_data(self):\n        self.handling_the_first_popup()\n        self.click_on_contact_us()\n        self.fill_values_to_first_name_field(self.valid_first_name)\n        self.fill_values_to_last_name_field(self.invalid_last_name)\n        self.fill_values_to_email_field(self.valid_email)\n        self.fill_values_to_phone_field(self.valid_phone)\n        self.fill_values_to_content_field(self.valid_message)\n        self.click_on_submit()\n>       assert self.verifying_success_message(self.success_message_xpath) == \"נא להזין את השם המשפחה הנכון\"\nE       AssertionError: assert 'הפרטים נקלטו בהצלחה' == 'נא להזין את השם המשפחה הנכון'\nE         - נא להזין את השם המשפחה הנכון\nE         + הפרטים נקלטו בהצלחה\n\nTests\\test_contactus.py:240: AssertionError"}, "description": "Test invalid last name", "steps": [{"name": "Opening the qa-trado website", "status": "passed", "start": 1677081697734, "stop": 1677081710941}], "start": 1677081697733, "stop": 1677081714359, "uuid": "b8f9df15-3b6f-4c91-a31e-b7698d337b81", "historyId": "c35b94a74766317b650fbfba3b46876f", "testCaseId": "c8a511c08d34cd0b6c5b4399de5a7e64", "fullName": "Tests.test_contactus.TestContactUs#test_all_correct_except_invalid_last_name_data", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_contactus"}, {"name": "subSuite", "value": "TestContactUs"}, {"name": "host", "value": "mike"}, {"name": "thread", "value": "18868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_contactus"}]}